packages/strapi-parts/src/Loader/Loader.stories.mdx

```diff
@@ -10,14 +11,27 @@ import { Loader } from './Loader';
 
 # Loader
 
-This is the doc of the `Loader` component
+Loaders indicate some data is currently loading where content usually takes place. They help to notify users that loading is ongoing.
 
-## Loader
+**Best** **practices**
 
-Description...
+- Once the content appears, the loader disappears.
+- If the trigger only concerns a portion of a page, the loader should be placed in that part.
+- Loaders should be scaled down if they aimed to be placed in a small portion of a page.
+- Use a loader whenever the wait time is to be longer than 1 second.
```

Comment:
This will be tough until React 18 is out. Meaning replacing every single way strapi is getting data from the server (and that's a lot)

Reply:
Let's remove it for now then :)